packages/scripts/scripts/build-blocks-manifest.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+const fs = require( 'fs' );
+const path = require( 'path' );
+const { sync: glob } = require( 'fast-glob' );
+const json2php = require( 'json2php' );
+const chalk = require( 'chalk' );
+
+/**
+ * Internal dependencies
+ */
+const { getArgFromCLI } = require( '../utils' );
+
+// Set default paths
+const defaultInputDir = 'build';
```

Comment:
Should the default be `build` or `src`? I can see why you chose `build`, but a counter argument would be that this means you always have to run `npm run build` _before_ you run `npm run build:blocks-manifest`.

To my knowledge, the `block.json` files are copied as is from `src` to `build`, so in that case I think `src` is a better default. WDYT?

Reply:
I was about to move it to `src` here, but I found that in order to keep the `npm run build:blocks-manifest` command working in the root, I'd have to look through 4 src directories:
```
./packages/blocks/src/
./packages/widgets/src/
./packages/block-library/src/
./packages/edit-widgets/src/
```
We don't have a way to specify multiple input dirs. It may be the `build->build` after is a useful pattern for monorepos?

Reply:
This one is interesting. When you use `wp-scripts build` then `block.json` files get copied from `src` to build and sometimes they even can get modified:

https://github.com/WordPress/gutenberg/blob/79940e0ca94cd814631e9a29f2d9178352246c70/packages/scripts/config/webpack.config.js#L314-L345

In effect, I think it's a valid approach to source the `block.json` from the `build` folder to put them in the `build` folder. In the long run, these probably should be an option on the `wp-scripts build --use-manifest`. However, we can get there in steps.

Reply:
That all makes sense, let's keep using `build` for now.